starter-code/main.js

```diff
@@ -0,0 +1,284 @@
+								console.log(result);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							}
+						});
+					break;
+					case "5":
+						db.collection('companies').find({
+								founded_year: 2004,
+								founded_month: {$gte: 4, $lte: 6}
+							},
+							{name: 1, founded_month: 1, founded_year: 1, _id: 0},
+							{sort: [["founded_month", "desc"], ["founded_day", "desc"]]}
+						).toArray((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							} else {
+								console.log(result);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							}
+						});
+					break;
+					case "6":
+						db.collection('companies').find({"offices.city": "Barcelona"}, {name: 1, _id: 0}).toArray((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							} else {
+								console.log(result);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							}
+						});
+					break;
+					case "7":
+						db.collection('companies').find({}, {name: 1, number_of_employees: 1, _id: 0})
+						.sort({number_of_employees: -1})
+						.limit(10).toArray((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							} else {
+								console.log(result);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							}
+						});
+					break;
+					case "8":
+						db.collection('companies').find({name: "Facebook"}).toArray((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							} else {
+								console.log(result);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							}
+						});
+					break;
+					case "9":
+						db.collection('companies').find({name: "Facebook"}, {number_of_employees: 1, _id: 0}).toArray((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							} else {
+								console.log(result);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							}
+						});
+					break;
+					case "10":
+						db.collection('companies').find({name: "Facebook"}, {"products.name": 1, _id: 0}).toArray((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							} else {
+								console.log(result);
+								rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
+							}
+						});
+					break;
+					case "11":
+						db.collection('companies').find(
+								{name: "Facebook", "relationships.is_past": false},
```

Comment:
You are using filter, there is no need to query for relationships.is_past (that won't work, anyway).